### PR TITLE
Use a fixed version of JHipster that is retrieving during generation

### LINKED ISSUE #48: Target jhipster version should be prompted
Main reason is that the generated package.json should use a fixed dependency version to generator-jhipster (see https://github.com/jhipster/generator-jhipster/issues/10415)

And also it would be a pain to update the template for each jhipster release (especially since generator-jhipster-blueprint is not often released),

### PR DESCRIPTION
**Pull request by murdos**
If fetching of version failed (e.g. no available network), prompt the user for the target version

Fixes #48

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "chalk": "2.4.1",
     "gulp-filter": "5.1.0",
     "mkdirp": "0.5.1",
+    "npm-api": "^1.0.0",
```
Comment (pascalgrimaud) [inline package.json]: shouldn't we use a fixed version instead ?

Comment (murdos) [reply package.json]: Oh, right! I just fixed it.

**Comment by pascalgrimaud** [comment]
@murdos : don't hesitate to ask me if you want a new release of blueprint

**Comment by pascalgrimaud** [comment]
Ok, I'll try to do a new release today

**Pull request merged**